product-description.md

```diff
@@ -11,18 +11,19 @@ If you are interested in exploring this product on GitHub, please click [here](h
 
 #### Data Sources
 All datasets are in both CSV and JSON format:
-- States current testing data (`states-current`)
-- States historical testing data (`states-daily`)
-- States specific resources (`states-info`)
-- US current testing data (`us-current`)
-- US historical testing data (`us-daily`)
-- COVID-19 related info from counties (`counties`)
-- Track URLs for regional specific data (`url`)
-- State website screenshots (`states-screenshots`)
-- COVID Track Project press (`press`)
+- States current testing data (`states_current`)
+- States historical testing data (`states_daily`)
+- States specific resources (`states_info`)
+- US current testing data (`us_current`)
+- US historical testing data (`us_daily`)
+- State website screenshots (`states_screenshots`)
```

Comment:
Are we changing all these filenames to have underscores instead of hyphens? If so, why?

Reply:
No, we are not changing the files names. The files are been, and will continuing to use, underscores.

Reply:
Hey @mwkaufman, quick followup. The `product-description.md` file was incorrect here. This product has been using underscores for its file names. I copied over the new version of the "Data Sources" section after adjusting the `README.md`, and didn't even notice the text had been incorrect here this whole time.